core/src/main/java/com/yahoo/ycsb/workloads/ClosedEconomyWorkload.java

```diff
@@ -0,0 +1,736 @@
+/**
+ * Copyright (c) 2010 Yahoo! Inc. All rights reserved.
```

Comment:
Please use Copyright (c) YCSB contributors. with the year this file was authored.